oak_attestation_verification/src/claims.rs

```diff
@@ -175,16 +177,103 @@ pub fn verify_validity_duration<T>(
 
 /// Checks that the given endorsement statement is a valid and has the correct
 /// claim type.
-pub fn validate_endorsement(
-    claim: &Statement<ClaimPredicate<EndorsementStatement>>,
-) -> Result<(), InvalidClaimData> {
+pub fn validate_endorsement(claim: &EndorsementStatement) -> Result<(), InvalidClaimData> {
     validate_claim(claim)?;
     if claim.predicate.claim_type != ENDORSEMENT_V2 {
         return Err(InvalidClaimData::ClaimType);
     }
     Ok(())
 }
 
+/// Assembles digests found in endorsement statement into a protocol buffer.
+pub fn get_digest(claim: &EndorsementStatement) -> anyhow::Result<HexDigest> {
+    if claim.subject.len() != 1 {
+        anyhow::bail!(
+            "expected a single endorsement subject, found {}",
+            claim.subject.len()
+        );
+    }
+
+    let mut digest = HexDigest {
```

Comment:
Could `HexDigest::default()` be used here?

Reply:
Done.